### PR TITLE
Add mobile app build flavor

### PR DESCRIPTION
**Pull request by m4dz**
This PR add a new build target for mobile app which introduces a `__TARGET__` compil var and use it to disable some chunks of code like the drawer.

### CODE REVIEW

**Comment by enguerran** [comment]
It's hard for me to review this code as I have never used cozy-bar.
I don't know where it is injected in cozy apps and how it is integrated in mobile versions.

**Comment by m4dz** [comment]
@enguerran here how to inject it in apps (for files-v3 in this case): https://github.com/cozy/cozy-files-v3/pull/121 :smile:

**Comment by enguerran** [comment]
Ok. so i guess it is ok to be merged.
@kosssi what do you think about?

File: config/webpack.target.mobile.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+const path = require('path')
+const webpack = require('webpack')
+const merge = require('webpack-merge')
+const {filename, production} = require('./webpack.vars')
+
+module.exports = merge(
+  require('./webpack.config.base.js'),
+  production ? require('./webpack.config.prod') : {},
```
Comment (enguerran) [inline config/webpack.target.mobile.js]: Why don't you add ` './webpack.config.dev'` as in [./config/webpack.target.split.js](./config/webpack.target.split.js)?

**Comment by enguerran** [comment]
I found a typo in https://github.com/cozy/cozy-bar/blob/11e33cc627bc6db10836162cc2d968cc82979f7b/src/lib/i18n.js#L18

```
console.warning()
```

**Comment by enguerran** [comment]
There is other heck: `The dict phrases for "{{.Locale}}" can't be loaded`
As it is not served by the cozy-stack.

**Comment by enguerran** [comment]
Neither does `{{.AppName}}` be replaced by the cozy-stack server.

File: config/webpack.target.mobile.js (hunk 1 above)
Comment (m4dz) [inline config/webpack.target.mobile.js]: Probably because it was in another PR, and I was waiting for it. I fix that.

**Comment by m4dz** [comment]
@enguerran thanks for the review, fixes incoming :)

**Comment by m4dz** [comment]
@enguerran I'm a little bit annoyed: I can't reproduce any of your issues :/… Can you explain to me how did your tests, so I can't find and fix them if needed? Thanks!

**Comment by m4dz** [comment]
It's now ok with https://github.com/cozy/cozy-files-v3/pull/121 and https://github.com/cozy/cozy-files-v3/pull/127

**Pull request merged**